scripts/build-sync-wheels

```diff
@@ -60,13 +62,18 @@ def main():
         subprocess.check_call(cmd)
         args.pkg_dir = git_clone_directory
     else:
+        args.pkg_dir = os.path.expanduser(args.pkg_dir)
         git_clone_directory = ""
         if not os.path.exists(args.pkg_dir):
             print(f"Project directory missing {args.pkg_dir}.")
             sys.exit(1)
 
     req_path = os.path.join(args.pkg_dir, args.requirements, "requirements.txt")
-    local_wheels = os.path.join(args.project, "wheels")
+    if args.project is not None:
+        project_name = args.project
+    else:
+        project_name = utils.project_name(Path(args.pkg_dir))
```

Comment:
I'd catch `RuntimeError` (or a more specific one maybe?) here and suggest using the `--project` option. I'm thinking that someone running into that error is very likely to not know about or have forgotten the option.

Something along the lines of: "If you're not creating a Python package, you can use the `--project` option."

Reply:
I adjusted the error message using your suggestion. (and fixed it when setup.py doesn't exist at all). I think this still accomplishes the goal without needing to add an extra try/except to both scripts.